public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -311,10 +311,9 @@ export function AnomalyResults(props: AnomalyResultsProps) {
             />
             <EuiText>
               <p>
-                Attempting to initialize the detector with historical data. This
-                initializing process takes approximately{' '}
-                {get(detector, 'detectionInterval.period.interval', 10)}{' '}
-                minutes.
+                Attempting to initialize the detector with historical data.
+                This initializing process takes approximately 1 minute if
```

Comment:
Is it possible to try and give any estimate if there is not enough consecutive data?

Reply:
That's another part of msg. Check the following in the same file:

```
const getInitProgressMessage = () => {
    return detector &&
      isDetectorInitializing &&
      !isHCDetector &&
      get(detector, 'initProgress.estimatedMinutesLeft')
      ? `The detector needs ${get(
          detector,
          'initProgress.estimatedMinutesLeft'
        )} minutes for initializing. If your data stream is not continuous, it may take even longer. `
      : '';
  };
```

Reply:
Ah I see, wasn't sure where it was.